tests/integration/config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    name: 'openfisca.org ',
+    baseURL: 'file://' + __dirname + '/../../public/index.html',
```

Comment:
This can't work in CI _if you use SauceLabs_. When you use SauceLabs, you are starting up testing machines in another cloud. SauceConnect opens a tunnel between your CI machine and the Sauce ones. This tunnel can forward HTTP(S) requests, but cannot forward `file` requests! You've set up everything right, but the machine on which the browser resides is simply not on the same filesystem as the machine on which you try to locate the path :wink:

Two solutions:

1. Use a lightweight server such as `python serve` or `live-server`.
2. Test locally rather than with SauceLabs, by starting up a `chromedriver` (which comes preconfigured [on Circle's images](https://circleci.com/docs/1.0/build-image-ubuntu-14.04-XL-922-9410082/#chromedriver) rather than installing and starting up `sc` (Sauce Connect). You'll probably need to set up the port so that Watai and ChromeDriver can communicate, as I don't think ChromeDriver's default is the same as Selenium’s, which Watai uses by default.

Considering all the work you've already done, and the added value of doing cross-browser testing, I think the less costly now is to use option 1. If no work had been done in setting the stack up, option 2 would probably have been cheaper 🙂